sdcm/sct_config.py

```diff
@@ -1889,7 +1889,8 @@ class SCTConfiguration(dict):
         'stress_cmd_lwt_d', 'stress_cmd_lwt_u', 'stress_cmd_lwt_i'
     ]
     ami_id_params = ['ami_id_db_scylla', 'ami_id_loader', 'ami_id_monitor', 'ami_id_db_cassandra', 'ami_id_db_oracle']
-    aws_supported_regions = ['eu-west-1', 'eu-west-2', 'us-west-2', 'us-east-1', 'eu-north-1', 'eu-central-1']
+    aws_supported_regions = ['eu-west-1', 'eu-west-2', 'us-west-2',
+                             'us-east-1', 'eu-north-1', 'eu-central-1', 'eu-west-3', 'ca-central-1']
```

Comment:
Do we have our AMIs copied to all these regions by default?

Reply:
Aha, I just saw you add a util for that.
I'm not sure we want to copy them to too many regions.

Reply:
the util is for those temporary one, not for scylla/monitoring images, that should be available on all region for releases

anyhow we should copy what's needed (and hope we have automatic cleanup for old images across all regions)